server/src-lib/Hasura/RQL/DDL/Remote/Validate.hs

```diff
@@ -38,7 +38,7 @@ data ValidationError
   | TableNotFound !QualifiedTable
   | TableFieldNonexistent !QualifiedTable !FieldName
   | ExpectedTypeButGot !G.NamedType !G.NamedType
-  | InvalidType !G.NamedType !T.Text
+  | InvalidType !G.GType!T.Text
```

Comment:
Going on record as saying Text in an error sum type is bad practice, but moving on.